dataclients/kubernetes/ingressdefinitions.go

```diff
@@ -65,6 +65,145 @@ func (s service) getTargetPortByValue(p int) (*definitions.BackendPort, bool) {
 	return nil, false
 }
 
+// There are [1..N] Kubernetes endpointslices created for a single Kubernetes service.
+// Kubernetes endpointslices of a given service can have duplicates with different states.
+// Therefore Kubernetes endpointslices need to be de-duplicated before usage.
+// The business object skipperEndpointSlice is a de-duplicated endpoint list that concats all endpointslices of a given service into one slice of skipperEndpointSlice.
+type skipperEndpointSlice struct {
```

Comment:
Lets move this into separate enpointslices.go file

Reply:
endpoints are also not in a separated file. I will split out both endpoints.go and endpointslices.go

Reply:
done